src/RevTracer.cc

```diff
@@ -0,0 +1,337 @@
+//
+// _RevTracer_cc_
+//
+// Copyright (C) 2017-2023 Tactical Computing Laboratories, LLC
+// All Rights Reserved
+// contact@tactcomplabs.com
+//
+// See LICENSE in the top level directory for licensing details
+//
+//
+
+#include <sstream>
+#include <iomanip>
+#include <string>
+
+#include "../include/RevTracer.h"
+#include "RevTracer.h"
+
+using namespace SST::RevCPU;
+using namespace std;
```

Comment:
Is this necessary? Seems like you prefix most of what you use from std namespace with `std::` anyways

Reply:
**NEVER** use `using namespace std;`!!!